### PR TITLE
Fix various inconsistencies

### PR DESCRIPTION
**Pull request by 2e3s**

### CODE REVIEW

File: spec/definitions/SubscriptionRenewalList.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
+type: array
+  items:
```
Comment (RomanGotsiy) [inline spec/definitions/SubscriptionRenewalList.yaml]: same here

**Comment by 2e3s** [comment]
CI fails. I think because RebillyReportsAPI aren't kept maintained unlike RebillyAPI.

**Comment by RomanGotsiy** [comment]
@2e3s try to rebase on master

File: spec/paths/reports@transactions.yaml [hunk 2]
```diff
@@ -2,8 +2,7 @@ get:
   tags:
     - Reports
   summary: Retrieve a transactions report
-  description: |
-    Retrieve a transactions report
+  description: Retrieve a transactions report
   parameters:
     - name: periodStart
       in: query
```
Comment (RomanGotsiy) [inline spec/paths/reports@transactions.yaml]: property `type` is required for parameter
add `type: string` here

**Comment by RomanGotsiy** [comment]
+ copy `mediaType` parameter you are referencing (copy from here https://github.com/Rebilly/RebillyAPI/blob/master/spec/swagger.yaml#L395)

**Pull request merged**